### ISSUE TITLE
Images functionality does not completely cover API

### ISSUE THREAD

**Issue by ihmccreery** [fog/fog-google#22]
Per #20 and testing I did yesterday, the `images` functionality is currently broken, as [the API](https://cloud.google.com/compute/docs/reference/latest/images#resource) has changed, but the codebase hasn't reflected that change.

**Comment by ihmccreery** [comment]
The current API expects *either*

- A `rawDisk` with a `source` URL to a GCS file and, optionally, a `container_type` and a checksum, which is somewhat consistent with the currently Fog functionality, or
- a `sourceDisk`: "URL of the The source disk used to create this image. This can be a full or valid partial URL. You must provide either this property or the rawDisk.source property but not both to create an image."

When given an invalid image URL, as in the Shindo tests, we get the following:

```
> Fog::Compute[:google].images.create(name: 'new-image', raw_disk: {source: 'some-random-url'})
Fog::Errors::Error: Invalid value for field 'rawDisk.source': 'some-random-url'.  The provided source location was not a valid Google Cloud Storage object reference.
```

**Comment by ihmccreery** [comment]
(cc @mouchtaris)

I've confirmed that I was wrong that the current implementation is broken.  In fact, it supports the `rawDisk` option above, as demonstrated in #23.

However, it does not support the `sourceDisk` option yet.  Hopefully I'll get that working sometime tomorrow.

**Comment by ihmccreery** [comment]
Since this is currently functional, I'm going to go ahead and close this for now.  I'll be revisiting total API coverage in the coming weeks, and this will be on that list.  @mouchtaris, if you need the `sourceDisk` functionality, let me know and I can try to get to it sooner, (or—even better—you can work on it yourself!).

**Comment by mouchtaris** [comment]
@ihmccreery Thanks a lot for your immediate effort! We are covered by the way fog-google works right now. I am making reports or pull requests for any other issue I get. Thanks!

**Comment by ihmccreery** [comment]
@mouchtaris In case you want it, I've released [v0.0.4](https://rubygems.org/gems/fog-google/versions/0.0.4) for the fix for #20.

Thanks for the bugs.  Keep 'em coming!  :wink:
